app/controllers/items_controller.rb

```diff
@@ -6,4 +6,20 @@ def index
   def show
     @item = Item.find(params[:id])
   end
+
+  def retire_item
```

Comment:
ah! yeah, that makes more sense on this controller XD nice work! I like the toggle method - clean